integration-tests/customhandler_test_utils.go

```diff
@@ -135,18 +153,32 @@ func systemLocalRow(cluster string, datacenter string, customPartitioner string,
 	if customPartitioner != "" {
 		partitionerValue = message.Column(customPartitioner)
 	}
+	if version >= primitive.ProtocolVersion3 {
```

Comment:
That was based on manual test with C* 2.0.0 running (via CCM). I have tested now with 2.0.17 and I see `rpc_address` present.  Since this is just mock for test, I guess we should test most pessimistic approach.

We do processing of `system.local` query analogical to Java 3.x driver. ZDM tries to parse `rpc_address` column. If it is not there ZDM has no choice than use address that was given to establish control connection.
```
addr, port, err := ParseRpcAddress(false, row, defaultPort)
if addr == nil {
    // could not resolve address from system.local table (e.g. not present in C* 2.0.0)
    addr, port, err = ParseEndpoint(ccEndpoint)
}
```

Reply:
Oh this just reminded me about https://github.com/datastax/zdm-proxy/issues/91 which is why we specifically say that we only support C* from 2.1.6 onwards (so we don't support 2.1.0-2.1.5).
If I understand correctly this means that now with this work we can close #91 AND we can now say that we support C* 2.1.0 and not just 2.1.6+ right?

Reply:
I just used origin = 2.0.17 and target = 2.1.1. Connected Java driver 3.x, created tables (with collection columns), inserted and queried rows.